### PR TITLE
Fix issue #1 by adding app.UseGlobalExceptionHandler() extension method

### PR DESCRIPTION
**Pull request by wintondeshong**
Fix issue #1 by adding extension methods to create result from IExceptionHandlerFeature as well as configure the IApplicationBuilder

Now applications just import the extension and call a single method to configure their application

```csharp
using AndcultureCode.CSharp.Web.Extensions;

app.UseGlobalExceptionHandler();
```

### CODE REVIEW

File: test/AndcultureCode.CSharp.Web.Tests/Extensions/IExceptionHandlerFeatureExtensionsTest.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+using AndcultureCode.CSharp.Testing.Extensions;
+using AndcultureCode.CSharp.Testing.Tests;
+using AndcultureCode.CSharp.Web.Extensions;
+using Microsoft.AspNetCore.Diagnostics;
+using Moq;
+using Shouldly;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace AndcultureCode.CSharp.Core.Tests.Unit.Extensions
```
Comment (brandongregoryscott) [inline test/AndcultureCode.CSharp.Web.Tests/Extensions/IExceptionHandlerFeatureExtensionsTest.cs]: I believe this namespace is incorrect; shouldn't it be `AndcultureCode.CSharp.Web.Tests.Extensions`?

File: test/AndcultureCode.CSharp.Web.Tests/Extensions/IExceptionHandlerFeatureExtensionsTest.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+{
+    public class IExceptionHandlerFeatureExtensionsTest : BaseUnitTest
+    {
+        #region Setup
+
+        public IExceptionHandlerFeatureExtensionsTest(ITestOutputHelper output) : base(output) { }
+
+        #endregion Setup
+
+
```
Comment (brandongregoryscott) [inline test/AndcultureCode.CSharp.Web.Tests/Extensions/IExceptionHandlerFeatureExtensionsTest.cs]: Minor, extra new line

File: test/AndcultureCode.CSharp.Web.Tests/Extensions/IExceptionHandlerFeatureExtensionsTest.cs (hunk 1 above)
Comment (wintondeshong) [reply test/AndcultureCode.CSharp.Web.Tests/Extensions/IExceptionHandlerFeatureExtensionsTest.cs]: snap, good catch

**Comment by brandongregoryscott** [comment]
@all-contributors add @wintondeshong for code, test, doc

**Pull request merged**